### DISCUSSION TITLE
./hrldas.exe runtime error

### DISCUSSION THREAD

**Discussion by hulumos** [NCAR/hrldas#162] [Ideas]
Hello, when running the Noah-MP single-point example, I encountered some issues. 
After successfully preparing the driver data and running, which generated several files, upon returning to the "run" directory and running "./hrldas.exe", an error occurred.
Here are some pictures：
![99BFC732@173BF633 336CE865](https://github.com/NCAR/hrldas/assets/141665972/db0cb551-8b96-4637-8447-91e1fff4458e)
![04D45A28@8C1F7F44 336CE865](https://github.com/NCAR/hrldas/assets/141665972/8f0d8889-7cca-4907-80e4-abd592bfe934)

**Comment by cenlinhe** [comment]
According to the error message, there is some formatting issue with your NoahmpTABLE.TBL file. Did you modify anything inside this table (maybe accidentally add some unexpected symbols)?

**Reply by hulumos** [reply]
Thank you for your response. In fact, I haven't modified the content of NoahmpTable.TBL, and I haven't even opened this file. Just now, I re-downloaded the code, including the NoahmpTable.TBL in the /hrldas-master/hrldas/run directory and the NoahmpTable.TBL file in the /hrldas-master/noahmp/parameters/ directory, and replaced them. After rerunning the model, I still received the same error.

**Reply by hulumos** [reply]
Currently, I replaced the NoahmpTable.TBL file in the /hrldas-master/noahmp/parameters/ directory with the one in the /hrldas-master/hrldas/run directory, and the code runs successfully. I want to express my gratitude to you once again.
